polysh/remote_dispatcher.py

```diff
@@ -24,12 +24,15 @@
 import termios
 import select
 import platform
+from argparse import Namespace
 
 from polysh.buffered_dispatcher import BufferedDispatcher
 from polysh import callbacks
 from polysh.console import console_output
 from polysh import display_names
 
+options = None  # type: Optional[Namespace]
```

Comment:
Yes, otherwise mypy doesn't know what Optional is. Pyflakes already added support to recognize this and not show those imports as unused: https://github.com/PyCQA/pyflakes/issues/247#issuecomment-310513633

Reply:
Hu, good point. Honestly all type comments that are still in should have gone into that type hinting patch, but they slipped through while rebasing. These won't hurt though, so let's just leave them in and merge the rest of the type hints once we have python3.5 in the relevant places.